build/x86_64_centos8/Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM registry.access.redhat.com/ubi8/ubi:latest
+FROM centos:8
 
-ADD https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
 ADD https://rpms.remirepo.net/RPM-GPG-KEY-remi2018 /etc/pki/rpm-gpg/RPM-GPG-KEY-remi.el8
-COPY centos8.repo remi.repo /etc/yum.repos.d/
+COPY remi.repo /etc/yum.repos.d/
```

Comment:
Do we still need this?

Reply:
Yes, the EPEL 8 repo didn't change, so some packages are still distributed from here.